client/tailscale/keys.go

```diff
@@ -71,10 +71,18 @@ func (c *Client) Keys(ctx context.Context) ([]string, error) {
 // CreateKey creates a new key for the current user. Currently, only auth keys
 // can be created. Returns the key itself, which cannot be retrieved again
 // later, and the key metadata.
-func (c *Client) CreateKey(ctx context.Context, caps KeyCapabilities) (string, *Key, error) {
+func (c *Client) CreateKey(ctx context.Context, caps KeyCapabilities, expirySeconds int64) (string, *Key, error) {
+
+	ninetydays := int64(90 * 24 * 60 * 60)
+
+	if expirySeconds > ninetydays {
```

Comment:
Recommend not checking the validity of the expiration parameter in the client: maybe someday support will be added for longer expirations. The API call will fail if the expiration is out of range.